README.md

```diff
@@ -249,3 +250,74 @@ Optionally you can switch GLSB back to round-robin mode
 ```shell script
 make init-round-robin
 ```
+## Metrics
+
+OhMyGLB generates [Prometheus][prometheus]-compatible metrics.
+Metrics endpoints are exposed via `-metrics` service in operator namespace and can be scraped by 3rd party tools:
+
+``` yaml
+spec:
+...
+  ports:
+  - name: http-metrics
+    port: 8383
+    protocol: TCP
+    targetPort: 8383
+  - name: cr-metrics
+    port: 8686
+    protocol: TCP
+    targetPort: 8686
+```
+
+Metrics can be also automatically discovered and monitored by [Prometheus Operator][prometheus-operator] via automatically generated [ServiceMonitor][service-monitor] CRDs , in case if [Prometheus Operator][prometheus-operator]  is deployed into the cluster.
+
+### General metrics
+
+[controller-runtime][controller-runtime-metrics] standard metrics, extended with OhMyGLB operator-specific metrics listed below:
+
+#### `healthy_records`
+
+Number of healthy records observed by OhMyGLB.
+
+Example:
+
+```yaml
+# HELP ohmyglb_gslb_healthy_records Number of healthy records observed by OhMyGLB.
+# TYPE ohmyglb_gslb_healthy_records gauge
+ohmyglb_gslb_healthy_records{name="test-gslb",namespace="test-gslb"} 6
+```
+
+#### `managed_hosts`
+
+Number of managed hosts observed by OhMyGLB.
+
+Example:
+
+```yaml
+# HELP ohmyglb_gslb_managed_hosts Number of managed hosts observed by OhMyGLB.
+# TYPE ohmyglb_gslb_managed_hosts gauge
+ohmyglb_gslb_managed_hosts{name="test-gslb",namespace="test-gslb",status="Healthy"} 1
+ohmyglb_gslb_managed_hosts{name="test-gslb",namespace="test-gslb",status="NotFound"} 1
+ohmyglb_gslb_managed_hosts{name="test-gslb",namespace="test-gslb",status="Unhealthy"} 2
```

Comment:
@somaritane a little bit confused here. Number of managed hosts is orthogonal to status(unless managed hosts are down to 0)  or do I miss something ?

Reply:
The metric is derived from ServiceHealth, it represents the amount of services with particular status, probably the naming is not correct, happy to consider better options.

Reply:
So currently Gslb Status looks like
```
  status:
    healthyRecords:
      app3.cloud.example.com:
      - 172.17.0.4
      - 172.17.0.5
      - 172.17.0.6
    managedHosts:
    - app1.cloud.example.com
    - app2.cloud.example.com
    - app3.cloud.example.com
    serviceHealth:
      app1.cloud.example.com: NotFound
      app2.cloud.example.com: Unhealthy
      app3.cloud.example.com: Healthy
```
So managed hosts == ingress host ohmyglb operates and serviceHealth is associated status... I will think of combined depicting name, seems to be not easy :)

Reply:
Yes, so we calculate how many ingress hosts are with certain status, and record this number  with "status" label accordingly. This can be aggregated in query to give amount of all ingress hosts within gslb etc.

Reply:
Just a side not: Looking at serviceHealth and managedHosts status properties, managedHosts looks kind of redundant, as from serviceHealth it is clear how many managed ingress hosts we have.

Reply:
@somaritane `ohmyglb_gslb_ingress_hosts_per_status_total` ? Inspired by https://prometheus.io/docs/practices/naming/

re `Status` content it needs to be streamlined I have it in mind for a while but never created an Issue - will do

Reply:
Looks good, just without `_total` suffix, as this metric is not accumulating count, it is [gauge](https://prometheus.io/docs/concepts/metric_types/#gauge), whose value can arbitrarily go up and down, metrics linter fails in case if `_total` suffix is used when naming Gauge and derived metric types.

Reply:
The metric is renamed to `ohmyglb_gslb_ingress_hosts_per_status`